central/main.go

```diff
@@ -887,6 +887,15 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 		})
 	}
 
+	if features.ComplianceEnhancements.Enabled() && features.ComplianceReporting.Enabled() {
+		customRoutes = append(customRoutes, routes.CustomRoute{
+			Route:         "/v2/compliance/scan/configurations/reports/download",
+			Authorizer:    user.With(permissions.Modify(resources.Compliance)),
+			ServerHandler: complianceScanSettings.NewDownloadHandler(),
+			Compression:   true,
+		})
```

Comment:
Shouldn't we expose this path anyway and return an error if the FFs are disabled? That could make the testing easier.

Reply:
🤔 I'm not sure. I think this is how it's done for VM.